readme.md

```diff
@@ -553,6 +553,7 @@ Non-synced local preferences are stored within `Alfred.alfredpreferences` under
 - [alfred-jira](https://github.com/colinf/alfred-jira) - Convert clipboard text between Markdown and Jira markup
 - [alfred-homebrew](https://github.com/vinkla/alfred-homebrew) - Search for macOS packages with Homebrew
 - [alfred-network-location-switch](https://github.com/abdul/alfred-network-location-switch) - Switch macOS network location
+- [alfred-cool](https://github.com/nguyenvanduocit/alfred-cool) - Find for cool words
```

Comment:
> Find cool words

You can also add tests easily with [alfy-test](https://github.com/samverschueren/alfy-test).

Reply:
Sure, I will

Reply:
@SamVerschueren Done

Reply:
Can you change the description as well to `Find cool words`? Looks better imo.

Reply:
sure, I am no very good at English.